.rubocop.yml

```diff
@@ -33,3 +33,6 @@ Style/TernaryParentheses:
 
 Style/VariableNumber:
   Enabled: false
+
+Lint/UselessAssignment:
```

Comment:
Why was this added?

Reply:
I added this for a test and forgot to remove it. I will remove it.